windowlist.h

```diff
@@ -1,6 +1,9 @@
+#include <X11/Xlib.h>
```

Comment:
> So the reasoning is that the header file is using Display and Window, from XLib, right?

Exactly. This is more important when you're developing a library than in this case, but it's a nice touch. Anyway I added it because my LSP was complaining  :smile: 

> However, doesn't this mean we should remove the include in main.c as well?

In general, it is good practice to always include the headers of an external library in each file it's being used. The idea behind that is if `windowlist.h` were to remove its dependency to `Xlib.h`, the program would still compile.  
Because `main` is not only using stuff from `windowlist`, but from `Xlib` as well.  
Anyway these things are always debated and I don't have an opinion :sweat_smile:. If you want we can remove it

Reply:
Okay, your explanation really helps.

So as long as it's deliberate that the include is removed from `windowlist.c`, I trust you and your LSP that it's a good change :smile: